lms/djangoapps/certificates/api.py

```diff
@@ -228,8 +229,13 @@ def certificate_downloadable_status(student, course_key):
         'is_unverified': True if current_status['status'] == CertificateStatuses.unverified else False,
         'download_url': None,
         'uuid': None,
+        'is_invalid_cert': False,
```

Comment:
This method also used by instructor [cert_generation_enabled](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/instructor/views/instructor_dashboard.py#L333).